src/LessMsi.Gui/Model/ApplicationSettings.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Xml.Serialization;
+
+namespace LessMsi.Gui.Model
+{
+	public class ApplicationSettings
+	{
+		static string applicationSettingsFile;
+		static ApplicationSettings applicationSettings;
+
+		public List<string> RecentFiles { get; set; } = new List<string>();
+
+
+		public void Save()
+		{
+			try
+			{
+				using (TextWriter writer = new StreamWriter(ApplicationSettingsFile))
+				{
+					XmlSerializer serializer = new XmlSerializer(typeof(ApplicationSettings));
+					serializer.Serialize(writer, this);
+				}
+			}
+			catch
+			{
+				// Should we nag the user?
+			}
+		}
+
+		static string ApplicationSettingsFile
+		{
+			get
+			{
+				if (applicationSettingsFile == null)
+				{
+					string directory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+					applicationSettingsFile = Path.Combine(directory, "lessmsi-gui-settings.xml");
+				}
+				return applicationSettingsFile;
+			}
+		}
+
+		public static ApplicationSettings Default
+		{
+			get
+			{
+				if (applicationSettings == null)
+				{
+					try
+					{
+						using (FileStream fs = new FileStream(ApplicationSettingsFile, FileMode.Open))
+						{
+							XmlSerializer serializer = new XmlSerializer(typeof(ApplicationSettings));
+							applicationSettings = (ApplicationSettings)serializer.Deserialize(fs);
+						}
+					}
+					catch

```

Comment:
As with my other comment above, I don't like catch handlers with no exception listed. Please be explicit about the exception we expect here and let the other ones bubble up so the user reports them. 

BTW: Specifically what I've seen happen is that you end up with some other weird behavior that you spend lots of time troubleshooting and and a catch like this (or even `catch (Exception...)`) is catching and hiding some exception that would have otherwise led you to the cause of an unrelated issue. Even more bizarre, is that C#/.NET will even catch unmanaged CRT exceptions with this type of catch handler without any exception listed, and that is super weird when that happens!